### DISCUSSION TITLE
Uso de docstring no projeto

### DISCUSSION THREAD

**Discussion by allangrds** [BrasilAPI/BrasilAPI#207] [Ideas]
O que acham de usarmos o JSDoc? Dessa forma conseguimos deixar o nosso código - que tem alguns comentários - documentado.

**Comment by RodriAndreotti** [comment]
Acho totalmente excelente, como diria Paulo Bonfá... rs
Acho importante o código estar documentado, para alguém que se aventure por ele.

**Comment by PauloGoncalvesBH** [comment]
Não tenho ideia formada sobre, mas se for adotado, que seja adotado junto com o `require-jsdoc`.
https://eslint.org/docs/rules/require-jsdoc
